tce/src/base/bem/BEMTester.cc

```diff
@@ -73,38 +73,37 @@ BEMTester::canAddComponentEncoding(
 
     int socketEncodings = field.socketEncodingCount();
     for (int i = 0; i < socketEncodings; i++) {
-	SocketEncoding& existingEnc = field.socketEncoding(i);
+        SocketEncoding& existingEnc = field.socketEncoding(i);
         int alignment = calculateAlignment(
             encodingWidth, existingEnc.socketIDWidth(), field);
-        int commonBits =
-	    commonBitCount(
-		encoding, extraBits, existingEnc.encoding(),
-		existingEnc.extraBits(), alignment);
-	if (commonBits == static_cast<int>(MathTools::bitLength(encoding))
-            + static_cast<int>(extraBits) || 
-            commonBits == existingEnc.socketIDWidth()) {
-	    return false;
-	}
+        int commonBits = commonBitCount(
+            encoding, extraBits, existingEnc.encoding(),
+            existingEnc.extraBits(), alignment);
+        if (commonBits == static_cast<int>(MathTools::bitLength(encoding))
+                + static_cast<int>(extraBits) || 
+                commonBits == existingEnc.socketIDWidth()) {
+            return false;
+        }
     }
 
     SourceField* sField = dynamic_cast<SourceField*>(&field);
     if (sField != NULL) {
-	int bridgeEncodings = sField->bridgeEncodingCount();
-	for (int i = 0; i < bridgeEncodings; i++) {
-	    BridgeEncoding& existingEnc = sField->bridgeEncoding(i);
-            int alignment = calculateAlignment(
-                encodingWidth, existingEnc.width(), field);
-	    int commonBits =
-		commonBitCount(
-		    encoding, extraBits, existingEnc.encoding(),
-		    existingEnc.extraBits(), alignment);
-	    if (commonBits == static_cast<int>(
-                    MathTools::bitLength(encoding)) + 
+    int bridgeEncodings = sField->bridgeEncodingCount();
+    for (int i = 0; i < bridgeEncodings; i++) {
+        BridgeEncoding& existingEnc = sField->bridgeEncoding(i);
+        int alignment = calculateAlignment(
+            encodingWidth, existingEnc.width(), field);
+        int commonBits =
+            commonBitCount(
+              encoding, extraBits, existingEnc.encoding(),
+              existingEnc.extraBits(), alignment);
```

Comment:
Misindentation here. Arguments of commonBitCount() call does not have indentation of 4 spaces but 2 instead.